exporters/trace/ocagent/src/test/java/io/opencensus/exporter/trace/ocagent/FakeOcAgentTraceServiceGrpcImpl.java

```diff
@@ -100,36 +105,67 @@ public void onCompleted() {}
   @Override
   public StreamObserver<CurrentLibraryConfig> config(
       StreamObserver<UpdatedLibraryConfig> updatedLibraryConfigStreamObserver) {
-    updatedConfigObserverRef.set(updatedLibraryConfigStreamObserver);
-    return currentConfigObserver;
+    synchronized (this) {
+      updatedConfigObserverRef.set(updatedLibraryConfigStreamObserver);
+      return currentConfigObserver;
+    }
   }
 
   @Override
   public StreamObserver<ExportTraceServiceRequest> export(
       StreamObserver<ExportTraceServiceResponse> exportTraceServiceResponseStreamObserver) {
-    return exportRequestObserver;
+    synchronized (this) {
```

Comment:
be consistent in using synchronized everywhere instead of synchronized (this)

Reply:
Good catch, fixed.